c/.gitignore

```diff
@@ -0,0 +1,15 @@
+# binaries
+hello
+
+# build
+*.[aod]
+
+# config
+compile_commands.json
+.pre-commit-config.yaml
+.cache
+
+# nix
+.direnv
+.envrc
```

Comment:
```suggestion
```
this can be commited, as long as it doesn't assume nix is installed. this can work for example
```
if has nix_direnv_version; then
  use flake
fi
```

Reply:
yeah this would be better removed